pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
As @ntoll says this is likely down to my Windows setup. (I assume it's
because he's worked from a zipped patch I sent over which would have
used my line-endings: my gitconfig has suitable eol configs set).

@ntoll, if you have the time to run a converter over the PR to smooth
out the eol diffs, feel free. Otherwise I'm happy to get to it myself
but it won't be today.